SonicWall/sonicwall-fw/ingest/parser.yml

```diff
@@ -0,0 +1,1655 @@
+          file.name: '{{parsed_event.message.fname}}'
+          file.mtime: "{{parsed_event.message.get('fileModification Time')}}"
+          file.path:  "{{parsed_event.message.filePath}}"
+          file.size: '{{parsed_event.message.fsize}}'
+          file.type:  "{{parsed_event.message.fileType}}"
+          host.domain: "{{parsed_event.message.get('deviceNt Domain')}}"
+          host.hostname: "{{parsed_event.message.shost or parsed_event.message.sourceHostName}}"
+          host.id: "{{parsed_event.message.get('device ExternalId')}}"
+          host.name: "{{parsed_event.message.dvchost or {parsed_event.message.get('deviceDns Domain')}}"
+          host.network.ingress.bytes: "{{parsed_event.message.in}}"
+          host.network.egress.bytes: "{{parsed_event.message.out}}"
+          http.request.referrer: "{{parsed_event.message.requestContext}}"
+          http.request.method: "{{parsed_event.message.requestMethod}}"
+          log.syslog.facility.name: "{{parsed_event.message.deviceFacility}}"
+
+      - set:
+          network: >
+            {
+              {% set list1 = parsed_event.message.proto.split('/') %}
+              "transport": "{{list1[0]}}",
+              {% if list1[1] and list1[1].isalpha() %}
+                "protocol": "{{list1[1]}}"
+              {% endif %}
+            }
+
+      - set:
+          observer.egress.interface.name: "{{parsed_event.message.deviceOutboundInterface or parsed_event.message.get('Device Outbound Interface')}}"
+          observer.ingress.interface.name: "{{parsed_event.message.deviceInboundInterface or parsed_event.message.get('deviceInbound Interface')}}"
+          observer.type: 'firewall'
+          process.entity_id: '{{parsed_event.message.appid}}'
+          process.name: "{{parsed_event.message.appName or parsed_event.message.get('deviceProcess Name')}}"
+          rule.name: '{{parsed_event.message.cs1}}'
+          service.name: "{{parsed_event.message.get('destination ServiceName')}}"
+          server.ip: "{{parsed_event.message.dvc}}"
+          source.bytes: '{{parsed_event.message.out}}'
+          source.domain: "{{parsed_event.message.get('sourceDns Domain')}}"
+          source.ip: '{{parsed_event.message.src or parsed_event.message.sourceAddress or parsed_event.message.srcV6}}'
+          source.mac: '{{parsed_event.message.smac}}'
+          source.nat.ip: "{{parsed_event.message.cs1Label or parsed_event.message.natSrcV6 or parsed_event.message.get('source Translated Address')}}"
+          source.nat.port: "{{parsed_event.message.snpt or parsed_event.message.get('source TranslatedPort')}}"
+          source.packets: '{{parsed_event.message.cn2Label}}'
+          source.port: '{{parsed_event.message.spt}}'
+          source.user.name: '{{parsed_event.message.suser}}'
+          user_agent.original: "{{parsed_event.message.get('requestClient Application')}}"
+          user.name: '{{parsed_event.message.susr}}'
```

Comment:
According to the tests, this field may hold a username with a user domain. I suggest splitting the value with a grok stage like [this example](https://github.com/SEKOIA-IO/intake-formats/blob/59859d1b7048a0a24f00b7bb82940b8dc9118fee/Azure/azure-windows/ingest/parser.yml#L20-L30)